src/ducks/settings/TagsSettings.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { tagsConn } from 'doctypes'
+import { useQueryAll, isQueryLoading } from 'cozy-client'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import TagsListSettings from 'ducks/settings/TagsListSettings'
+
+const TagsSettings = () => {
+  const { data: tags, ...tagsQueryRest } = useQueryAll(tagsConn.query, tagsConn)
+  const isLoading = isQueryLoading(tagsQueryRest) || tagsQueryRest.hasMore
```

Comment:
You can have more status: 
- loaded 
- isLoading 
- failed 

The two first are handled ATM, but here, you don't deal with the "failed" state.